homeassistant/components/fyta/sensor.py

```diff
@@ -0,0 +1,171 @@
+"""Summary data from Fyta."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Final
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FytaCoordinator, FytaEntity
+
+
+@dataclass(frozen=True)
+class FytaSensorEntityDescription(SensorEntityDescription):
+    """Describes Fyta sensor entity."""
+
+    value_fn: Callable[[str | int | float], str | int | float | datetime] = (
+        lambda value: value
+    )
+
+
+SENSORS: Final[list[FytaSensorEntityDescription]] = [
+    FytaSensorEntityDescription(
+        key="plant_name",
+        translation_key="fyta_plant_name",
+    ),
```

Comment:
Isn't this already something for the device name?

Reply:
Device names can change on boot or on runtime with extra code. If the user changes the device name, the name shown won't change but then it's their problem.